### PR TITLE
feat(tsdb): delete time series data

### PR DESCRIPTION
**Pull request by DuanKuanJun**

### CODE REVIEW

File: src/client/src/tscSql.c [hunk 1]
```diff
@@ -416,6 +416,13 @@ int taos_affected_rows(TAOS_RES *tres) {
   return pSql->res.numOfRows;
 }
 
+int taos_affected_tables(TAOS_RES *tres) {
```
Comment (hjxilinx) [inline src/client/src/tscSql.c:419]: why is this API needed?

File: src/util/inc/tqueue.h [hunk 2]
```diff
@@ -47,6 +47,9 @@ void      *taosAllocateQitem(int size);
 void       taosFreeQitem(void *item);
 int        taosWriteQitem(taos_queue, int type, void *item);
 int        taosReadQitem(taos_queue, int *type, void **pitem);
+// special type search Qitem
+int        taosSearchQitem(taos_queue, int type, void **pitem);
```
Comment (hjxilinx) [inline src/util/inc/tqueue.h:51]: search in queue?

File: src/client/src/tscSql.c (hunk 1 above)
Comment (DuanKuanJun) [reply src/client/src/tscSql.c:419]: Deleted 1000 row(s) from 1000 table(s) (0.331877s) .  1000 tables can be got by taos_affected_tables() api

File: src/util/inc/tqueue.h (hunk 2 above)
Comment (DuanKuanJun) [reply src/util/inc/tqueue.h:51]: yes , need remove Qitem from queue after async delete all rows from disk.

**Comment by DuanKuanJun** [comment]
this is test branch , close PR

**Pull request closed without merging**